record_indexer/index_templates/rikolti_template.py

```diff
@@ -0,0 +1,240 @@
+import sys, os
+import json
+import boto3
+import botocore
+import requests
+import copy
+
+'''
+    Create OpenSearch index template for rikolti
+    https://www.elastic.co/guide/en/elasticsearch/reference/7.9/index-templates.html
+    https://www.elastic.co/guide/en/elasticsearch/reference/7.9/indices-component-template.html
+'''
+
+ENDPOINT = os.environ.get('RIKOLTI_ES_ENDPOINT')
+AUTH = ('rikolti', os.environ.get('RIKOLTI_ES_PASS'))
+
+def main():
+
+    properties = get_properties()
+
+    # TODO add aliases, version, _meta, priority
+    payload = {
+        "index_patterns": ["rikolti*"],
+        "template": {
+            "settings": {
+                "number_of_shards": 1,
+                "analysis": {
+                    "analyzer": {
+                        "keyword_lowercase_trim": {
+                            "tokenizer": "keyword",
+                            "filter": ["trim", "lowercase"]
+                        }
+                    }
+                }
+            },
+            "mappings": {
+                "dynamic": "strict",
+                "properties": properties
+            }
+        }
+    }
+
+    # create the API request
+    url = os.path.join(ENDPOINT, "_index_template/rikolti_template")
+
+    headers = {
+        "Content-Type": "application/json"
+    }
+
+    # create index template
+    r = requests.put(url, headers=headers, data=json.dumps(payload), auth=AUTH)
+    r.raise_for_status()
+    print(r.text)
+
+
+def get_properties():
```

Comment:
given that this just returns a static JSON structure, does this really need to be a function? I feel like you've maybe overthought this a bit.

Reply:
See commit ad082dd3fc87bd64ea08c959629a048b0822ea0e

Reply:
OK yeah, I see your point. I was using python to generate the JSON for me, but there's no need to memorialize that in the repo. It's easier to see what's going on here.